### PR TITLE
[RDAPI-23542] - Added MySQL 8 driver information

### PR DESCRIPTION
**Pull request by lpinto-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

The existing documentation suggests the use of MySQL connector v5.1.47 when using Maria DB database. However, this version is not compatible with Maria DB but a lower version such as 5.1.29 is. It is also recommended that MySQL 8 is used in conjunction with v5.1.49 of the connector. 

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by andreamussap** [comment]
@lpinto-axway There's something wrong with this PR. It cannot have 78 commits. Maybe you've started from one branch and tried to change to another, between master and developmar21 ?? I don't know but please check this. I won't be possible to merge it like this.

**Comment by cgardineraxway** [comment]
> @lpinto-axway There's something wrong with this PR. It cannot have 78 commits. Maybe you've started from one branch and tried to change to another, between master and developmar21 ?? I don't know but please check this. I won't be possible to merge it like this.

The problem is **lpinto-axway wants to merge 78 commits into master from RDAPI-23542** this should be wants to merge RDAPI-23452 into developmar21.

**Comment by lpinto-axway** [comment]
That's odd, it showed me one commit before I submitted it. Let me try again.

**Comment by lpinto-axway** [comment]
Revised PR - https://github.com/Axway/axway-open-docs/pull/1734
